foundations/installations/installations.md

```diff
@@ -502,4 +506,4 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- To learn more about WSL2, you can visit [the official Microsoft documentation on installing WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
```

Comment:
We should probably just link directly to https://learn.microsoft.com/en-us/windows/wsl rather than the install page specifically. People will have already installed it by this stage, now they are just looking for more info.